plugin.video.spurs-tv/resources/lib/youtube.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
```

Comment:
This isn't needed for Krypton, as it ships with python 2.7 on all platforms.
Python 2.7 has native support for the print function.

Please tell me if you want to remove it, or keep it.

Reply:
I think it is needed to use the print function in 2.7.

https://docs.python.org/2.7/library/functions.html#print
> This function is not normally available as a built-in since the name print is recognized as the print statement. To disable the statement and use the print() function, use this future statement at the top of your module:
> 
> from __future__ import print_function